### ISSUE TITLE
[6.3.0] Empty YouTube tracks

### ISSUE THREAD

**Issue by kiritokatklian** [Androz2091/discord-player#1731]
**Describe the bug**
Searching for a song using the YouTube extractor returns an empty array of tracks. This still works fine on 6.2.1, but stopped working in 6.3.0

**To Reproduce**

Update to 6.3.0 then use the following code:
```js
const { YouTubeExtractor } = require('@discord-player/extractor')

const tracks = await player.search("Example", {
		requestedBy: interaction.user,
		searchEngine: YouTubeExtractor.identifier
	})
		.then(x => x.tracks)
		.catch(e => console.error(e))

console.log(tracks) // []
```

**Expected behavior**
Receive an array of tracks.

**Screenshots**
n/a

**Please complete the following information:**
 - Node Version: 19.9.0
 - Discord Player Version: 6.3.0
 - Discord.js Version: 14.10.2

**Additional context**
- play-dl Version: 1.9.2

Tried replacing `play-dl` with `ytdl-core`, specifically from [this issue](https://github.com/fent/node-ytdl-core/issues/1216), but that didn't fix it.

The only way I got it working again is by downgrading back to 6.2.1, switching back to `play-dl`, and using `QueryType.YOUTUBE` instead of `YouTubeExtractor.identifier`.

**Comment by Crocross** [comment]
Did you load the extractor too?
```js
await player.extractors.register(YoutubeExtractor, {});
```

**Comment by skdhg** [comment]
The `searchEngine: YouTubeExtractor.identifier` is invalid. Use
```js
searchEngine: `ext:${YouTubeExtractor.identifier}`
```

**Comment by kiritokatklian** [comment]
@Crocross yes I've tried both loadDefault() and registering only YouTubeExtractor. Both removed the warning about no extractors being registered, but both gave empty tracks too.

@skdhg Thanks! Prepending `ext:` fixed the issue. I didn't see it in the [guide](https://discord-player.js.org/docs/guides/stream-sources) on the docs website, so I'm guessing this is a bug or at least not the intended way of using it. If not, then it'd be handy to update the docs to include it to avoid confusion in the future.

Thanks again!

**Comment by skdhg** [comment]
> I didn't see it in the [guide](https://discord-player.js.org/docs/guides/stream-sources) on the docs website, so I'm guessing this is a bug or at least not the intended way of using it. If not, then it'd be handy to update the docs to include it to avoid confusion in the future.

thanks for pointing this out @kiritokatklian 

There's a mistake in the guide. It should be

```diff
+ import { SoundCloudExtractor } from '@discord-player/extractor';
+
+ const result = await player.search(query, {
+    // always use soundcloud extractor
-    searchEngine: SoundCloudExtractor.identifier
+    searchEngine: `ext:${SoundCloudExtractor.identifier}`
+});
```
